app/src/main/java/ciserver/PipelineUpdateRequest.java

```diff
@@ -28,8 +36,9 @@ class RequestPath {
     }
 }
 
-// Body parameters. If this class is needed for another
-// type of post request later on, we could add more.
+/**
+ * The RequestBody contains the body parameters of the PipelineUpdateRequest.
+ **/
 class RequestBody {
     String state;
     String target_url;
```

Comment:
You can use `@SerializedName("target_url") String targetUrl` in order to comply with Java naming.